src/validation.cairo

```diff
@@ -290,19 +290,11 @@ fn validate_coinbase(block: @Block, total_fees: u64, block_height: u32) -> Resul
     assert(*tx.inputs[0].previous_output.txid == Default::default(), 'txid should be 0');
 
     // Validate the outputs' amounts
-    // Sum up the total amount of all outputs
-    // and also add the outputs to the UtreexoSet.
+    // Sum up the total amount of all outputs of the coinbase transaction
     let mut total_output_amount = 0;
-
-    for txs in *block
-        .txs {
-            for output in *txs
-                .outputs {
-                    total_output_amount = total_output_amount + *output.value;
-                    //TODO add outputs to UtreexoSet
-
-                };
-        };
+    for output in *tx.outputs {
+        total_output_amount = total_output_amount + *output.value;
```

Comment:
total_output_amount += *output.value would be more concise

Reply:
That's true I had seen it too, and forgot to change it